x-pack/plugins/security/server/lib/privileges/privilege_action_registry.js

```diff
@@ -11,6 +11,19 @@ import { buildPrivilegeMap } from './privileges';
 import { getClient } from '../../../../../server/lib/get_client_shield';
 import { equivalentPrivileges } from './equivalent_privileges';
 
+export async function registerPrivilegesIfNecessary(server, plugin, xpackInfo) {
+  const { allowRbac } = xpackInfo.feature('security').getLicenseCheckResults();
+
+  if (allowRbac) {
+    try {
+      await registerPrivilegesWithCluster(server);
+    } catch (e) {
+      plugin.status.red(`Unable to register privileges`);
+      server.log(['security', 'error'], `Unable to register privileges with cluster: ${e}`);
+    }
+  }
+}
+
 export async function registerPrivilegesWithCluster(server) {
```

Comment:
We shouldn't need to export this anymore.

Reply:
I left this exported because it's easier to unit test that way. and I think there's value in having dedicated tests for this, rather than going through `registerPrivilegesIfNecessary` every time